plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/resources/org/eclipse/che/plugin/languageserver/ide/languageserver.css

```diff
@@ -49,5 +49,9 @@
 }
 
 .codeassistant-detail {
-    color: #326EED;
+    color: completionPopupItemSubtitleTextColor;
+}
+
+.codeassistant-highlight {
+    color: completionPopupItemHighlightTextColor;
 }
```

Comment:
@TylerJewell Sorry, but I don't understand your request. This part is formatted like the rest of the file.